src/DoctrineWriter.php

```diff
@@ -262,47 +287,50 @@ protected function disableLogging()
      */
     protected function reEnableLogging()
     {
-        $config = $this->entityManager->getConnection()->getConfiguration();
+        //TODO: do we need to add support for MongoDB logging?
+        if (!($this->objectManager instanceof \Doctrine\ORM\EntityManager)) return;
```

Comment:
It's not a user tries to do illegal actions. The logger for SQL is disabled to speed up the write operations. Honesty, I don't know how to disable the logging for MongoDB. That's why this functionality is supported for Doctrine ORM only.